### PR TITLE
Push cordova instructions

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues:

* [AEROGEAR-9353](https://issues.jboss.org/browse/AEROGEAR-9353)

This PR add miss steps for push on Cordova

### CODE REVIEW

**Comment by danielpassos** [comment]
@b1zzu I think those are the only miss from the [AEROGEAR-9353](https://issues.jboss.org/browse/AEROGEAR-9353). Can you take a look?

File: modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc [hunk 1]
```diff
@@ -40,6 +40,8 @@ Cordova::
 --
 // end::excludeDownstream[]
 `src/mobile-services.json`
+
+NOTE: To run the Cordova app on the Android device you also need to download the `google-services.json` from Firebase and put it on the root folder of your project.
```
Comment (b1zzu) [inline modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: I think it should be also important to mention that this line:
```
<platform name="android">
  <resource-file src="google-services.json" target="app/google-services.json" />
</platform>
```
should be added to the config.xml 

An alternative could be to add the link to the PhoneGap push plugin documentation: https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/INSTALLATION.md

**Comment by b1zzu** [comment]
@danielpassos I'm going throw all the push documentation to see if there are still old parts and I found the documentation on how to deploy Push server in Openshift: https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/push/provisioning-the-push.adoc is it still valid?!

**Comment by b1zzu** [comment]
@danielpassos also here: https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/push/handling-push-notifications.adoc
image

we could set js for the code highlights ( ```js ) and add from where PushRegistraiont is imported:
```js
import { PushRegistration } from "@aerogear/push";
...
```

File: modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc (hunk 1 above)
Comment (finp) [reply modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: I think this is important info (both from @danielpassos  and @b1zzu ), but I feel that it's landing in the wrong place in the docs. 
BTW, do we rely on phonegap push plugin, or is it just that the aerogear plug in uses same reference?

Comment (b1zzu) [reply modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: @finp yes we fully relly on phonegap push plugin, the aerogear push plugin is a wrapper around the phonegap one

Comment (b1zzu) [reply modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: As feedback: the first time I followed the documentation I took quite a lot to figure out that under the neath there was phonegap, and once I discovered it I managed to fix issued like the google-services.json by following the documentation on the phonegap repository

Comment (danielpassos) [reply modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: @finp Can you point me a better place to put that info?

**Comment by danielpassos** [comment]
@b1zzu I have updated it with your suggestions. Can you give a second eyes on it?

File: modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc (hunk 1 above)
Comment (danielpassos) [reply modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc]: ping @finp

**Comment by danielpassos** [comment]
@finp Ok. I have removed the commit which removes the 'provisioning push' section.

**Comment by danielpassos** [comment]
@finp @b1zzu Anything else should I handle?

**Pull request merged**